### DISCUSSION TITLE
Videos for VC intro, how to contribute

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#154] [Ideas]
@Virtual-Coffee/ideas-team We've thought about doing videos to help create the VC experience clearer, and thanks to @Aman83770 for bringing this up again. 

I'd like to do some brainstorming for the content we think we should cover, how we should cover it, and then we can break it down into a process and categories.

Right now, I like the idea of having an intro to our community and our tools, and then how to contribute. 

For some of these things, we might be able to grab already created resources and find out where to put them. For example, I bet slack has some way of showing how to use slack and add yourself to a channel.

So drop your ideas for the videos as a reply to this discussion and/or links to already created resources that might work as well.

### Extra context
![image](https://user-images.githubusercontent.com/34313413/120230697-e5ca1b80-c21d-11eb-993d-b0e2a0b00562.png)

**Comment by iritush** [comment]
I like the idea of these videos. Would the be link to on the website?
I would add that I think it will be good to have a corresponding written resources for those who prefer to read instructions. It is also easier to refer back to text when you re looking for something specific that you forgot like syntax for example vs looking for that information in the video. Happy to help writing these up.

**Reply by BekahHW** [reply]
I think the intro could be on the site, and then for new members, we could have something set up in slack.

I made a couple of looms yesterday to walk through how to use some of the things on github for our contributors. Nothing huge, but trying to provide the video option in addition to documentation. For example:

https://www.loom.com/share/14959671463a4edbbe80edbc45f35606

**Reply by danieltott** [reply]
We could also add one or both to the VC Slack bot App Home!
